hieradata/dev/wso2/wso2is/pattern-2/default.yaml

```diff
@@ -110,29 +122,29 @@ wso2::sso_authentication:
   consumer_service_url: https://is.dev.wso2.org:9443/acs
 
 wso2::master_datasources:
- wso2_config_db:
-   name: WSO2_CONFIG_DB
-   description: The datasource used for config registry
+ wso2_reg_db:
+   name: WSO2_REG_DB
+   description: The datasource used for shared registry
    driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
-   url: jdbc:mysql://192.168.100.1:3306/WSO2_CONFIG_DB?autoReconnect=true
+   url: jdbc:mysql://192.168.100.1:3306/is540reg?autoReconnect=true&amp;useSSL=false
```

Comment:
Wouldn't it be better to keep database names consistent, with regards to naming and case?